exs/mundo_1/readme.md

```diff
@@ -1,5 +1,44 @@
+| 021        | C#                  | C++                  | [Python](./python/021.py) | Java                    | [JavaScript](./javascript/021.html) | [Go](./golang/021.go)  | [Ruby](./ruby/021.rb) |
+| 022        | C#                  | C++                  | [Python](./python/022.py) | Java                    | [JavaScript](./javascript/022.js)   | [Go](./golang/022.go)  | [Ruby](./ruby/022.rb) |
+| 023        | C#                  | C++                  | [Python](./python/023.py) | Java                    | [JavaScript](./javascript/023.js)   | [Go](./golang/023.go)  | [Ruby](./ruby/023.rb) |
+| 024        | C#                  | C++                  | [Python](./python/024.py) | Java                    | [JavaScript](./javascript/024.js)   | [Go](./golang/024.go)  | [Ruby](./ruby/024.rb) |
+| 025        | C#                  | C++                  | [Python](./python/025.py) | Java                    | [JavaScript](./javascript/025.js)   | [Go](./golang/025.go)  | [Ruby](./ruby/025.rb) |
+| 026        | C#                  | C++                  | [Python](./python/026.py) | Java                    | [JavaScript](./javascript/026.js)   | Go  | [Ruby](./ruby/026.rb) |
+| 027        | C#                  | C++                  | [Python](./python/027.py) | Java                    | [JavaScript](./javascript/027.js)   | Go  | [Ruby](./ruby/027.rb) |
+| 028        | C#                  | C++                  | [Python](./python/028.py) | Java                    | [JavaScript](./javascript/028.js)   | Go  | [Ruby](./ruby/028.rb) |
+| 029        | C#                  | C++                  | [Python](./python/029.py) | Java                    | [JavaScript](./javascript/029.js)   | Go  | [Ruby](./ruby/029.rb) |
+| 030        | C#                  | C++                  | [Python](./python/030.py) | Java                    | [JavaScript](./javascript/030.js)   | Go  | [Ruby](./ruby/030.rb) |
+| 031        | C#                  | C++                  | [Python](./python/031.py) | Java                    | [JavaScript](./javascript/031.js)   | Go  | [Ruby](./ruby/031.rb) |
+| 032        | C#                  | C++                  | [Python](./python/032.py) | Java                    | [JavaScript](./javascript/032.js)   | Go  | [Ruby](./ruby/032.rb) |
+| 033        | C#                  | C++                  | [Python](./python/033.py) | Java                    | [JavaScript](./javascript/033.js)   | Go  | [Ruby](./ruby/033.rb) |
+| 034        | C#                  | C++                  | [Python](./python/034.py) | Java                    | [JavaScript](./javascript/034.js)   | Go  | [Ruby](./ruby/034.rb) |
+| 035        | C#                  | C++                  | [Python](./python/035.py) | Java                    | [JavaScript](./javascript/035.js)   | Go  | [Ruby](./ruby/035.rb) |
+=======
```

Comment:
Ocorreu algum conflito interno em seu ambiente que adicionou os seguintes identificadores:
-  `<<<<<<< HEAD` no inicio da tabela
-  `=======`  no final da tabela
- também o hash do conflito no final do arquivo `>>>>>>> 00ee6c5183542ac732d1cb2bb0d5afdfd6a333e0`

Corrija isso removendo os identificadores necessários, deixe o padrão conforme as tabelas existente e adicione os links para os novos exercícios adicionados. 

Caso tenha dúvida, basta verificar o [markdown da atual tabela no repositório](https://raw.githubusercontent.com/Coding-in-community/exercicios-CeV/master/exs/mundo_1/readme.md).